### PR TITLE
Fix data verification for Table Writer Fuzzer

### PR DESCRIPTION
**Pull request by kewang1024**
When verifying data, now it uses the inputVector to construct split thus the 
comparison is always true.
Change to construct the split from files written.

### CODE REVIEW

File: velox/exec/fuzzer/FuzzerUtil.h [hunk 1]
```diff
@@ -32,6 +32,11 @@ std::vector<Split> makeSplits(
     const std::string& path,
     const std::shared_ptr<memory::MemoryPool>& writerPool);
 
+/// Create splits from files in a directory. Don't support nested directory.
+/// TODO: Add support for nested directory, need to parse directory information
```
Comment (xiaoxmeng) [inline velox/exec/fuzzer/FuzzerUtil.h:36]:
```
///
/// TODO: ...
```

File: velox/exec/fuzzer/FuzzerUtil.cpp [hunk 2]
```diff
@@ -50,6 +51,16 @@ std::vector<Split> makeSplits(
   return splits;
 }
 
+std::vector<Split> makeSplits(const std::string& directory) {
```
Comment (xiaoxmeng) [inline velox/exec/fuzzer/FuzzerUtil.cpp:54]: Why not you pass a list of files to make splits? Thanks!

File: velox/exec/fuzzer/FuzzerUtil.h (hunk 1 above)
Comment (kewang1024) [reply velox/exec/fuzzer/FuzzerUtil.h:36]: It seems the same as above?

File: velox/exec/fuzzer/FuzzerUtil.cpp (hunk 2 above)
Comment (kewang1024) [reply velox/exec/fuzzer/FuzzerUtil.cpp:54]: So when we have partitioned table, the directory would look like
/p0=0/p1=0/file1
/p0=1/p1-0/file2
...

In such cases, we need to be able to parse out partition information and put it in the split schema

File: velox/exec/fuzzer/FuzzerUtil.cpp [hunk 3]
```diff
@@ -50,6 +51,16 @@ std::vector<Split> makeSplits(
...
+  std::vector<Split> splits;
+  auto fileSystem = filesystems::getFileSystem("/", nullptr);
+  for (auto const& p : std::filesystem::directory_iterator{directory}) {
+    VELOX_CHECK(!p.is_directory(), "");
```
Comment (xiaoxmeng) [inline velox/exec/fuzzer/FuzzerUtil.cpp:58]:
```
VELOX_CHECK(!p.is_directory());
```

**Review by xiaoxmeng — APPROVED** [review]

**Pull request closed without merging**